.eslintrc.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
 
     'react/jsx-curly-spacing': ['error', { when: 'never', children: true }],
     indent: ['error', 2, { SwitchCase: 1 }],
-    'linebreak-style': ['error', 'windows'],
+    'linebreak-style': ['error', 'unix'],
```

Comment:
Can I ask what is this change doing?

Reply:
I adjust eslint.js line endings to CRLF for local development on Windows to prevent linting errors. Before pushing, I revert them to LF to maintain consistency with our project's Unix line endings.